Contributing.md

```diff
@@ -0,0 +1,120 @@
+disclosure which should be sent privately to the contact address
+listed in the [Code of Conduct](CoC).
+
+Committers may direct you to another repository, ask for additional
+clarifications, and add appropriate metadata before the issue is
+addressed.
+
+Please be courteous and respectful.  Every participant is expected
+to follow the [Code of Conduct](CoC)).
+
+# Contributions
+
+Any change to resources in this repository must be through pull
+requests (PRs). This applies to all changes to documentation, code,
+binary files, etc. Even long term committers and TC members must
+use pull requests.
+
+In general, the submitter of a PR is responsible for making changes
+to the PR.  Any changes to the PR can be suggested by others in the
+PR thread (or via PRs to the PR), but changes to the primary PR
+should be made by the PR author.  In order to merge a PR, it must
+satisfy both of these two conditions:
+
+  1. have been open for 24 hours
+  2. have two approvals
+
+Pull requests should sit for at least 24 hours to ensure that
+contributors in other timezones have time to review. Consideration
+should also be given to weekends and other holiday periods to ensure
+active committers all have reasonable time to become involved in
+the discussion and review process if they wish.
+
+In order to encourage involvement and review, we require at least
+two explicit approvals from committers that are not the PR author.
+
+The default for each contribution is that it is accepted once no
+committer has an objection, and the above two requirements are
+satisfied.  During review committers may also request that a specific
+contributor who is more versed in a particular area gives an approval
+before the PR can be merged. There is no additional "sign off"
+process for contributions to be merged. Once all issues brought by
+committers are addressed it can be merged by any committer.
+
+In the case of an objection being raised in a pull request by another
+committer, all involved committers should seek to arrive at a
+consensus by way of addressing concerns being expressed by discussion,
+compromise on the proposed change, or withdrawal of the proposed
+change.
+
+If a contribution is controversial and committers cannot agree about
+how to get it merged or if it should merge, then it should be escalated
+to the TC. TC members should regularly discuss pending contributions
+in order to find a resolution. It is expected that only a small
+minority of issues be brought to the TC for resolution and that
+discussion and compromise among committers be the default resolution
+mechanism.
+
+Exceptions to the above are minor typo fixes or cosmetic changes
+that don't alter the meaning of a document.  Those edits can be
+made via a PR which only requires a single approval (not two, and
+not open for 24 h) to be merged.
+
+
+# Becoming a Committer
+
+All contributors who get a non-trivial contribution merged should
```

Comment:
I think we should say "if they so wish" or something to that effect, there may be situations when significant contributors may not want to participate at a higher level or situations where the TC does not want to approve or invite a contributor.

Reply:
That is very reasonable.  I guess that this mode of 'automatically' becoming a committer is a simple application of the rules, rather than requiring someone to ask.  This is very equitable, but may end up with us adding a bunch of one-time committers, or those that don't want those powers, as you point out.  Look for a new commit to this PR branch in an hour or so, and I'll update this, see if you like the changed language.

Reply:
As a Committer doesn't have obligations but only added rights, i don't understand what the danger is here?

Reply:
There's no particular 'danger' that I can think of.  From the organization side, there is a potential for a lot of adding of committers that only make a single contribution and then are never seen again, and it just becomes a 'management' issue to keep track of this list of those with commit-access and determine who really needs that access and who doesn't at some later date.

I suspect that in practice, this difference is like splitting hairs, since the TC determines who is made a Collaborator in either case.  It is just in the PSO case, that Collaborator-ship is given solely on the basis of code contribution, and nothing else, and there are also no exceptions.  In this modified case, the TC is allowed to consider other factors when admitting a Collaborator (including the wishes of a particular contributor--if they don't want write-access).  I think this language gives a little more latitude to the TC and the committers.

Reply:
Good points, thanks.